src/main/java/com/minecolonies/coremod/entity/pathfinding/pathjobs/AbstractPathJob.java

```diff
@@ -731,6 +732,11 @@ else if (onALadder(node.parent, node.parent, pos))
                 p.setOnLadder(true);
             }
 
+            if (next != null)
+            {
+                next.previous = p;
```

Comment:
this was null for no reason, making it useable

Reply:
We have not been using it.

Reply:
I would rather have the "next" variant than previous, but previous is already part of vanilla
usage: lookup more points in row without calling getPoint(index)